### DISCUSSION TITLE
Greetings From AFFiNE!

### DISCUSSION THREAD

**Discussion by Yipei-Operation** [toeverything/AFFiNE#84] [General]
Hello! We built an open-source privacy first Notion/Miro alternative! We call it Affine. 
It is not only a block docs, but also a visual learning/collaboration tool merged together. In short, it is a next-gen semantic knowledge base that put your planning, sorting and creating of content together.
If you like this open source project, please help us by clicking the GitHub star, which is crucial for the future development of AFFiNE. As we have just open-sourced it, it would be greatly appreciated if you could provide us with some review or feedback.
If there is anything that we can help with making AFFiNE more available to others, please let us know!

Here is our blog: https://affine.pro/blog  (Design, engineering, and everything else about AFFiNE)
Real-time QA is here: https://discord.gg/yz6tGVsf5p
Here is our Github(as you can see right now): https://github.com/toeverything/AFFiNE (If you like this open-source project, please help us by clicking the GitHub star 🌟）
Other Community/Social media link:
* Discord: https://discord.gg/Arn7TqJBvG
* Twitter:[ https://twitter.com/AffineOfficial](https://twitter.com/AffineOfficial)
* Medium：https://medium.com/@affineworkos
We hope you can have a nice time with AFFiNE!

**Comment by FranzAudio** [comment]
This has unlimited potential. Great stuff, keep it up and open source, wish you all the best.

**Reply by Yipei-Operation** [reply]
thanks a lot! Any suggestion/improvements/bug submission/ feature request is always welcomed at any time!

**Comment by saleemds** [comment]
Hello,
Any update on when the stable desktop / laptop version will be online?

The blog link does not seem to be working

thanks

**Reply by Yipei-Operation** [reply]
Hey, Thanks for the question~! I have updated our latest link for all of our community channels, and welcome to check our blog via https://affine.pro/blog and check anything via https://community.affine.pro/home
Next week we will launch our first beta version, and hope we can launch our first stable version before June 9th.
Any question welcome to contact us or ask here at any time~!

**Comment by saleemds** [comment]
Hello,

Thank you for the quick response. I appreciate it.

I am sure you are all very busy but a simple sample page with a note, todo
list and Kanban plus other features that you think are meaningful for task
management would be awesome. It doesn’t have to be dynamic, a static one
can just give some idea of the possibilities.

It’s really great to have a open source product of this type; I’ve spent
hours searching for one to suit my needs.

Wishing you all the best

**Reply by Yipei-Operation** [reply]
Hi, thanks for the suggestion! We plan to update our Github readme with the sample page within our next 6-week-iteration and hope we can let users know more user cases of AFFiNE, for now, can check our blog https://affine.pro/blog and our twitter to know more this kind of things~

**Comment by yordyi** [comment]
Good！

**Comment by fallsdevil** [comment]
Hey, I would like to know how I expose as an image the selected items in whiteboard frame mode? I know you have some export options, but they only work for normal text pages, and that's not what I want. Currently I use TLDRAW/EXCALIDARW ONLINE to be able to write down my mental maps, selecting them appears "export as PNG or SVG" and activates/disable the transparent background option. And I need this function because I use these images as a reference in Flashcards, and if the affine has this function, I migrate for it ... If at least you have an estimate of if this is in progress, in planning, I would thank you for know that this function will be implemented.
